### PR TITLE
Created sample email rule

### PR DESCRIPTION
**Pull request by elitegoodguy**
### Status 

Replace this: email, additional documentation as a sample rule

### Description

Sample Rule for email pack

### Checklist (tick everything that applies)

- [ ] Metadata: pack.yaml, icon, structure (required for new packs)
- [ ] Version bump (required for changed packs)
- [ ] Code linting (required, can be done after the PR checks)
- [ ] [Tests](https://docs.stackstorm.com/development/pack_testing.html) (not required but really recommended)

### CODE REVIEW

**Comment by elitegoodguy** [comment]
Take it easy on me first github PR

File: packs/email/rules/sample.emailsample.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+uid: 'rule:email:Sample'
```
Comment (Kami) [inline packs/email/rules/sample.emailsample.yaml]: uid is actually optional (it's automatically inferred from other fields), you can remove it from the resource metadata file.

Comment (Kami) [inline packs/email/rules/sample.emailsample.yaml]: Besides that, LGTM :+1:

File: packs/email/rules/sample.emailsample.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+tags: []
+type:
+  ref: standard
+  parameters:
+enabled: false
+name: Sample
+trigger:
+  ref: email.imap.message
```
Comment (Kami) [inline packs/email/rules/sample.emailsample.yaml]: Same here, only `type` should be required

**Comment by Kami** [comment]
I removed the attributes which are not needed when loading rule from disk (I assume you created and dumped it using an API), added description and sorted the attributes so they follow more standard order and merged PR into master - 667433b6f57587b83eeb3dae6114e8e103ba4964.

Thanks again!

**Pull request merged**